src/Lykke.Service.PayInvoicePortal/Views/Shared/_Layout.cshtml

```diff
@@ -129,6 +129,11 @@
                             </div>
                             <div class="dropdown__container">
                                 <ul class="dropdown__nav">
+                                    @if (!string.IsNullOrWhiteSpace(User.GetName()))
+                                    {
+                                        <li><a href="javascript:;" style="color:#000;cursor:default;">@User.GetName().ToUpper()</a></li>
```

Comment:
This is weird link. Why javascript:; instead of javascript:void(0)? Why this is a link without any action? May be I miss something.

Reply:
This is because of styling, as we don't have special for UserName for the moment - this is a link to look as others ;)
regarding anchor onclick event doing nothing:
javascript:; vs javascript:void(0) - returns the same and more a choise for readability
`
typeof(eval('javascript:void(0)'))
"undefined"
typeof(eval('javascript:;'))
"undefined"
`